README.md

```diff
@@ -1,8 +1,11 @@
 # Android Boilerplate
+<img src="https://raw.githubusercontent.com/SachseInTheHub/android-boilerplate/develop/android-ustwo.png" width="400">
```

Comment:
@SachseInTheHub You'll want this to be: https://raw.githubusercontent.com/ustwo/android-boilerplate/develop/android-ustwo.png which will be correct when you merge the branch. This prevents any issues arising from you deleting your fork